mrtt-ui/package.json

```diff
@@ -34,14 +34,26 @@
     "build": "react-scripts build",
     "test": "react-scripts test  --passWithNoTests",
     "eject": "react-scripts eject",
-    "lint": "eslint ./src/**/*.{js,jsx,tsx,ts}",
-    "lint:fix": "eslint './src/**/*.{js,jsx,tsx,ts}' --fix",
-    "prettier": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
+    "lint": "eslint './src/**/*.{js,jsx,tsx,ts}' --no-error-on-unmatched-pattern",
+    "lint:fix": "eslint './src/**/*.{js,jsx,tsx,ts}' --fix --no-error-on-unmatched-pattern",
+    "prettier": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
+    "storybook": "start-storybook -p 6006 -s public",
+    "build-storybook": "build-storybook -s public"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
       "react-app/jest"
+    ],
+    "overrides": [
+      {
+        "files": [
+          "**/*.stories.*"
+        ],
+        "rules": {
+          "import/no-anonymous-default-export": "off"
+        }
+      }
```

Comment:
just some auto modifications from the storybook installer, and fixing up linting scripts. (file pattern needed single quotes)